src/main/java/com/kodilla/ecommercee/service/CartService.java

```diff
@@ -46,13 +47,13 @@ public void deleteItem(Long itemId) {
         itemRepository.deleteById(itemId);
     }
 
```

Comment:
W public Cart createCart(Cart cart) w nawiasach nadal jest niepotrzebne Cart cart. Nigdzie tego carta nie wykorzystujemy, po prostu pusty nawias createCart() -> to samo poprawić w controlerze. Od strony Postmana nic nie wysyłamy,